pkg/collector/corechecks/snmp/internal/report/report_metrics.go

```diff
@@ -140,6 +140,10 @@ func (ms *MetricSender) reportColumnMetrics(metricConfig checkconfig.MetricsConf
 				tmpTags := common.CopyStrings(tags)
 				tmpTags = append(tmpTags, metricConfig.StaticTags...)
 				tmpTags = append(tmpTags, getTagsFromMetricTagConfigList(metricConfig.MetricTags, fullIndex, values)...)
+				if isInterfaceTableMetric(symbol.OID) {
```

Comment:
The only tag that seems to be present across all of them and populated is like you mentioned `interface`. I do worry a bit though if customers add a tag called interface themselves to another metric that's somehow unrelated we could start inadvertently tagging these metrics.

Florian and I were discussing perhaps some way for customers to configure in the profile some way of doing this so perhaps a future iteration of this will do away with the explicit check entirely in favor of going based off the profiles.

Those are the two other solutions I see that make sense at the moment.